src/storage/storage.ts

```diff
@@ -134,6 +161,26 @@ export class Storage implements IStorage {
     return this.store.getAddressAtIndex(index);
   }
 
+  /**
+   * Get the address public key, if not available derive from xpub
+   * @param {number} index
+   * @async
+   * @returns {Promise<string>} The public key DER encoded in hex
+   */
+  async getAddressPubkey(index: number): Promise<string> {
+    const addressInfo = await this.store.getAddressAtIndex(index);
```

Comment:
chore: We have no coverage on the new `getAddressPubkey` method. I feel this is important to validate the new feature.

Reply:
Added tests